js/genform.js

```diff
@@ -46,6 +46,9 @@ function generate(filename, field_array){
     // Converts camel case like 'abcDefGhi' into
     // the format 'Abc-Def-Ghi'
     function camelToHyphen(camelCaseWord) {
+        // introduce the special case for CSAF (short term solution)
+        if (camelCaseWord === "CSAF") {return "CSAF"}
```

Comment:
This looks good, just needs this change I think:
```suggestion
        if (camelCaseWord === "csaf") {return "CSAF"}
```

Reply:
@joker314, the CSAF field is all caps in the regitry: https://www.iana.org/assignments/security-txt-fields/security-txt-fields.xhtml.